### PR TITLE
[MM-20479] Migrate 'components/post_view/commented_on_files_message' module and associated tests to TypeScript

### PR DESCRIPTION
**Pull request by jakubnovak998**
#### Summary
- Rename the files to their associated TypeScript extensions (ie. js to ts, jsx to tsx)
- Update any components using those to the correct imports
- Fix any errors generated by the TypeScript compiler. You can run make check-types to display any errors.

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-server/issues/15462

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @jakubnovak998 for the contribution! It looks like the automated tests are failing, would you be open to help take a look? If you have any questions, let us know, happy to help :)

**Review by nevyangelova — APPROVED** [review]

**Comment by nevyangelova** [comment]
Thanks @jakubnovak998 thanks for the PR 🎉  LGTM minus the linter issues ;)

File: components/post_view/commented_on_files_message/commented_on_files_message.tsx [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 
-        /*
-         * An array of file metadata for the parent post
-         */
-        fileInfos: PropTypes.arrayOf(PropTypes.object),
-    }
+    /*
+     * An array of file metadata for the parent post
+     */
+    fileInfos?: Array<any>
```
Comment (sudheerDev) [inline components/post_view/commented_on_files_message/commented_on_files_message.tsx:17]: Can you check if `Array<FileInfo>` is appropriate?

**Review by sudheerDev — CHANGES_REQUESTED** [review]

**Comment by sudheerDev** [comment]
@jakubnovak998 Thanks for the contribution, Can you also update the snapshots for tests?

**Comment by hanzei** [comment]
Hey @jakubnovak998, I just wanted to check if you have any questions about the feedback above.

**Comment by jfrerich** [comment]
Hi @jakubnovak998!  Thanks for your contribution!  Do you have any questions or need any help completing this PR?

**Comment by devinbinnie** [comment]
Closing this PR as inactive, @jakubnovak998 let us know if you still plan on finishing this up.

**Pull request closed without merging**